tests/testthat/test-summary-log.R

```diff
@@ -1,5 +1,7 @@
 context("Test creating summary logs")
 
+suppressPackageStartupMessages(library(dplyr))
```

Comment:
my preference would be to directly namespace the pull rather than globally load dplyr if we do it, or just simplify the code to not need pull (as convenient as it is for quick writing

```
filter(sum_df2, run == "1001") %>% pull(minimization_terminated)
```

to just

```
filter(sum_df2, run == "1001")$minimization_terminated
```

Reply:
example in #445 for consideration

Reply:
> my preference would be to directly namespace the pull rather than globally load dplyr if we do it

Sure, that's fine by me.  As noted in my initial message ("Rather than qualify those calls [*], follow what test-param-labels.R does and just load dplyr"), my decision to load was pretty much based on the fact that this same line is already elsewhere in the tests.